data/mods/Magiclysm/Spells/druid.json

```diff
@@ -1403,5 +1403,116 @@
     "//": "EoC to summon because summoned comestibles don't have timers",
     "id": "EOC_DRUID_GOODBERRY",
     "effect": [ { "u_spawn_item": "item_druid_goodberry", "count": 1, "suppress_message": true } ]
+  },
+  {
+    "id": "druid_create_temporary_spring",
+    "type": "SPELL",
+    "name": "Fountain of the Depths",
+    "description": "Call to the waters beneath the earth and bring them to the surface, creating a natural spring.  The waters will subside when the spell's duration ends.",
+    "flags": [ "CHANNELING_SPELL", "CONCENTRATE", "SILENT", "VERBAL" ],
+    "valid_targets": [ "self" ],
+    "spell_class": "DRUID",
+    "difficulty": 9,
+    "max_level": 15,
+    "effect": "effect_on_condition",
+    "effect_str": "EOC_DRUID_TEMPORARY_SPRING",
+    "shape": "blast",
+    "min_duration": 30000,
+    "max_duration": 255000,
+    "duration_increment": 15000,
+    "energy_source": "MANA",
+    "base_casting_time": 9000,
+    "base_energy_cost": 750,
+    "extra_effects": [ { "id": "eoc_channeling_setup", "hit_self": true } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DRUID_TEMPORARY_SPRING",
+    "effect": [
+      {
+        "if": {
+          "u_query_tile": "around",
+          "target_var": { "global_val": "druid_temporary_spring_location" },
+          "message": "Select natural earth:"
+        },
+        "then": {
+          "if": { "map_terrain_with_flag": "DIGGABLE", "loc": { "global_val": "druid_temporary_spring_location" } },
+          "then": [
+            {
+              "u_transform_radius": 0,
+              "ter_furn_transform": "ter_druid_create_spring",
+              "target_var": { "global_val": "druid_temporary_spring_location" }
+            },
+            {
+              "u_transform_radius": 60,
+              "ter_furn_transform": "ter_druid_remove_spring",
+              "target_var": { "global_val": "druid_temporary_spring_location" },
+              "time_in_future": {
+                "math": [ "( (u_spell_level('druid_create_temporary_spring') * 150) + 300 ) * ( channeling_proficiency_modifier() )" ]
+              }
+            }
+          ],
+          "else": { "u_message": "You can only call up a spring on natural earth." }
+        },
+        "else": { "u_message": "Canceled" }
+      }
+    ]
+  },
+  {
+    "id": "druid_turn_tree_into_treant",
+    "type": "SPELL",
+    "name": "Awakening of the Forest Guardian",
+    "description": "Grant a nearby tree awareness and mobility, letting it shake free from the soil and rise up to crush your enemies.  It will root itself again when the spell expires.\n\nYou must be <color_yellow>standing next to a tree</color> for the spell to take effect.",
```

Comment:
If it dies while a treat does it leave behind a fallen tree trunk?

Reply:
Unfortunately no--I don't think there's a way to distinguish between the spell running out and dying in combat for the purposes of death effects.